### PR TITLE
[New Rule] Scheduled Task Execution at Scale via GPO

### PR DESCRIPTION
**Pull request by w0rk3r**
## Summary

Solves #1604 

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/windows/privilege_escalation_group_policy_scheduled_task.toml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+```
+"""
+references = [
+    "https://github.com/atc-project/atc-data/blob/master/docs/Logging_Policies/LP_0025_windows_audit_directory_service_changes.md",
+    "https://github.com/atc-project/atc-data/blob/f2bbb51ecf68e2c9f488e3c70dcdd3df51d2a46b/docs/Logging_Policies/LP_0029_windows_audit_detailed_file_share.md",
+    "https://labs.f-secure.com/tools/sharpgpoabuse",
+    "https://twitter.com/menasec1/status/1106899890377052160",
+    "https://github.com/SigmaHQ/sigma/blob/master/rules/windows/builtin/win_gpo_scheduledtasks.yml"
+]
+risk_score = 73
```
Comment (Samirbous) [inline rules/windows/privilege_escalation_group_policy_scheduled_task.toml:74]: maybe risk_score should be set to Medium (cuz still need to check the task content/action before marking it as high/80% suspicious).

Comment (w0rk3r) [reply rules/windows/privilege_escalation_group_policy_scheduled_task.toml]: Done

**Review by Samirbous — APPROVED** [review]

File: etc/non-ecs-schema.json [hunk 2]
```diff
@@ -22,5 +22,12 @@
   },
   "logs-windows.*": {
     "powershell.file.script_block_text": "text"
+  },
+  "logs-system.*": {
```
Comment (brokensound77) [inline etc/non-ecs-schema.json:26]: these need to go under winlogbeat

File: rules/windows/privilege_escalation_group_policy_scheduled_task.toml [hunk 3]
```diff
@@ -0,0 +1,116 @@
+[metadata]
+creation_date = "2021/11/08"
+maturity = "production"
+updated_date = "2021/11/08"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects the modification of Group Policy Object Attributes to execute a scheduled task in the objects controlled by the GPO.
```
Comment (brokensound77) [inline rules/windows/privilege_escalation_group_policy_scheduled_task.toml:9]:
```suggestion
Detects the modification of Group Policy Object attributes to execute a scheduled task in the objects controlled by the GPO.
```

File: etc/non-ecs-schema.json (hunk 2 above)
Comment (w0rk3r) [reply etc/non-ecs-schema.json:31]: Added

File: rules/windows/privilege_escalation_group_policy_scheduled_task.toml [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Privilege Escalation", "Active Directory"]
+timestamp_override = "event.ingested"
+type = "query"
+
+query = '''
+(event.code: "5136" and winlog.event_data.AttributeLDAPDisplayName:("gPCMachineExtensionNames" or "gPCUserExtensionNames") and 
+winlog.event_data.AttributeValue:(*CAB54552-DEEA-4691-817E-ED4A4D1AFC72* and *AADCED64-746C-4633-A97C-D61349046527*)) or
+(event.code: "5145" and winlog.event_data.ShareName: "\\\\*\\SYSVOL" and winlog.event_data.RelativeTargetName: *ScheduledTasks.xml and
+(message: WriteData or winlog.event_data.AccessList: *%%4417*))
```
Comment (brokensound77) [inline rules/windows/privilege_escalation_group_policy_scheduled_task.toml:86]:
```suggestion
(event.code: "5136" and winlog.event_data.AttributeLDAPDisplayName:("gPCMachineExtensionNames" or "gPCUserExtensionNames") and 
   winlog.event_data.AttributeValue:(*CAB54552-DEEA-4691-817E-ED4A4D1AFC72* and *AADCED64-746C-4633-A97C-D61349046527*)) 
or
(event.code: "5145" and winlog.event_data.ShareName: "\\\\*\\SYSVOL" and winlog.event_data.RelativeTargetName: *ScheduledTasks.xml and
  (message: WriteData or winlog.event_data.AccessList: *%%4417*))
```

**Review by brokensound77 — APPROVED** [review]
✔️

**Pull request merged**